_includes/docs/latest/setup-script-credentials.md

```diff
@@ -1,6 +1,5 @@
 Terraform uses an IAM Service Account to deploy and configure resources on behalf of the user.  The Service Account and required APIs can be setup automatically with a provided script on the 
 [Forseti Terraform Github repository](https://github.com/forseti-security/terraform-google-forseti/blob/master/helpers/setup.sh). 
-The Service Account and required APIs can also be configured manually by reviewing the [Requirements](https://forsetisecurity.org/docs/latest/setup/install.html#requirements) to deploy Forseti with Terraform.
```

Comment:
The Forseti on GKE installation steps references these setup-script files, and now the manual steps to create the service account and enable required APIs will not show up on the GKE steps. Can we add that back in, perhaps just add back into the forseti on gke page?

Reply:
Both Forseti on-GCE and Forseti on-GKE installation pages are referencing the `setup-script-credentials.md` which has the instructions. I have updated the setup script so manual steps to create the service account will show up.